wpsc-core/wpsc-includes.php

```diff
@@ -81,3 +81,11 @@
 
 // Cron
 require_once( WPSC_FILE_PATH . '/wpsc-includes/cron.php' );
+
+// WP-CLI support
+if ( defined( 'WP_CLI' ) && WP_CLI && version_compare( phpversion(), '5.3', '>=' ) ) {
+	require_once( WPSC_FILE_PATH . '/wpsc-includes/wpsc-wp-cli-category.php');
+	WP_CLI::add_command( 'wpsc-category', 'WPSC_WP_CLI_Category_Command' );
+	require_once( WPSC_FILE_PATH . '/wpsc-includes/wpsc-wp-cli-product-tag.php');
+	WP_CLI::add_command( 'wpsc-product-tag', 'WPSC_WP_CLI_Product_Tag_Command' );
+}
```

Comment:
As we iterate, it's probably going to make more sense to just replace this all with a require_once of a `wpsc-wp-cli.php` type of file, which will include all the sub-commands.